openquake/commonlib/nrml_examples/fragm_c.xml

```diff
@@ -1,60 +1,39 @@
-<?xml version='1.0' encoding='utf-8'?>
-<nrml xmlns="http://openquake.org/xmlns/nrml/0.4">
-
-    <!-- The fragility function format applies to the entire fragility model -->
-    <fragilityModel format="continuous">
-        <description>Fragility model for Pavia (continuous)</description>
-
-        <!-- limit states apply to the entire fragility model -->
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It makes sense. Currently there is automatic test for the conversion, it will probably be added in a different pull request.